package.json

```diff
@@ -24,7 +24,7 @@
     "format": "prettier --write \"**/*.+(js|jsx|json|yml|yaml|css|md|vue)\"",
     "start": "react-app-rewired start",
     "build": "react-app-rewired build",
-    "test": "react-app-rewired test --verbose --coverage --watchAll=false",
+    "test": "react-app-rewired test --verbose --coverage --watchAll=true",
```

Comment:
watchAll should be changed to false, otherwise testing will never finish and checks will fail